CleanProFinder/CleanProFinder.Mobile/ViewModels/ServiceUserEditPremiseViewModel.cs

```diff
@@ -0,0 +1,110 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CleanProFinder.Mobile.Services;
+using CleanProFinder.Shared.ServiceResponseHandling;
+using CommunityToolkit.Mvvm.Input;
+using CleanProFinder.Shared.Dto.Premises;
+
+namespace CleanProFinder.Mobile.ViewModels;
+
+[QueryProperty(nameof(PremiseId), nameof(PremiseId))]
+public partial class ServiceUserEditPremiseViewModel : ObservableObject
+{
+    private readonly IDialogService _dialogService;
+    private readonly IUserPremiseService _userPremisesService;
+
+    public ServiceUserEditPremiseViewModel(IDialogService dialogService, IUserPremiseService userPremisesServic)
+    {
+        _dialogService = dialogService;
+        _userPremisesService = userPremisesServic;
+        IsEditing = false;
+    }
+
+    private string _premiseId;
+
+    public string PremiseId
+    {
+        get => _premiseId;
+        set
+        {
+            SetProperty(ref _premiseId, value);
+            LoadPremise(value);
+        }
+    }
+
+    [ObservableProperty]
+    private string _address;
+
+    [ObservableProperty]
+    private string _square;
+
+    [ObservableProperty]
+    private string _description;
+
+    [ObservableProperty]
+    private bool _isEditing;
+
+    [RelayCommand]
+    private async void StartEditing()
+    {
+        IsEditing = true;
+    }
+
+    private async void LoadPremise(string premiseId)
+    {
+        ServiceResponse<OwnPremiseFullInfoDto> response =
+           await _userPremisesService.GetServiceUserPremiseFullInfoAsync(new Dictionary<string, object> { { "premiseId", premiseId } });
```

Comment:
Move dictionary initialization to a separate line

Reply:
Fixed